src/jsMain/kotlin/app/softwork/bootstrapcompose/Navbar.kt

```diff
@@ -0,0 +1,174 @@
+ * @param togglerAttrs Additional attributes to set on the toggler button
+ * @param attrs Additional attributes to set on the Navbar
+ * @param brand Composable implementing the brand elements
+ * @param navItems navigation items, normally comprised of NavbarLink and NavbarDropDown menu items.
+ */
+@Composable
+public fun Navbar(
+    breakpoint: Breakpoint = Breakpoint.Medium,
+    fluid: Boolean = false,
+    containerBreakpoint: Breakpoint? = null,
+    colorScheme: Color = Color.Light,
+    backgroundColor: Color = Color.Primary,
+    toggler: Boolean = true,
+    togglerLeft: Boolean = true,
+    togglerAttrs: AttrsBuilder<HTMLButtonElement>.() -> Unit = { },
+    attrs: AttrsBuilder<HTMLElement>.() -> Unit = { },
+    brand: (@Composable () -> Unit),
+    navItems: (@Composable () -> Unit)
+) {
+    Navbar(
+        breakpoint,
+        fluid,
+        containerBreakpoint,
+        colorScheme,
+        backgroundColor,
+        attrs
+    ) {
+        if (!togglerLeft) {
+            brand()
+        }
+
+        if (toggler) {
+            val targetId = "toggler${UUID().toString()}"
+            Toggler(
+                target = targetId,
+                controls = targetId,
+                togglerAttrs
+            )
+            NavbarCollapse(targetId) {
+                NavbarNav {
+                    navItems()
+                }
+            }
+        } else {
+            NavbarNav {
+                navItems()
+            }
+        }
+
+        if (togglerLeft) {
+            brand()
+        }
+    }
+}
+
+
+/**
+ * Bootstrap navbar-collapse component to be used with the NavbarToggler.
+ * @param id The element id. This value should also be used as the target parameter with a NavbarToggler.
+ */
+@Composable
+public fun NavbarCollapse(
+    id: String,
+    attrs: AttrsBuilder<HTMLDivElement>.() -> Unit = {},
+    content: ContentBuilder<HTMLDivElement>? = null
+) {
+    Div(attrs = {
+        classes(BSClasses.collapse, BSClasses.navbarCollapse)
+        attr("id", id)
+        attrs()
+    }) {
+        content?.invoke(this)
+    }
+}
+
+/**
+ * Bootstrap navbar-nav component to be used within a Navbar.
+ */
+@Composable
+public fun NavbarNav(
+    attrs: AttrsBuilder<HTMLDivElement>.() -> Unit = {},
+    links: ContentBuilder<HTMLDivElement>? = null
+) {
+    Div(attrs = {
+        classes(BSClasses.navbarNav)
+        attrs()
+    }) {
+        links?.invoke(this)
+    }
+}
+
+
+/**
+ * Bootstrap nav-link component.
+ */
+@Composable
+public fun NavbarLink(
+    active: Boolean,
+    attrs: AttrsBuilder<HTMLAnchorElement>.() -> Unit = {},
+    disabled: Boolean = false,
+    link: String? = null,
+    content: ContentBuilder<HTMLAnchorElement>? = null
+) {
+    A(attrs = {
+        classes(BSClasses.navLink)
+        if (active) {
+            classes(BSClasses.active)
+            attr("aria-current", "page")
+        }
+        attr("disabled", disabled.toString())
```

Comment:
This will produce `<a class="nav-link" disabled="true"></a>`, not `<a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Disabled</a>`